visualization/FairnessDashboard/src/Controls/IntroTab.tsx

```diff
@@ -15,15 +15,75 @@ export class IntroTab extends React.PureComponent<IIntroTabProps> {
         return (
             <Stack style={{ height: '100%' }}>
                 <div className={styles.firstSection}>
-                    <Text className={styles.firstSectionTitle} block>
-                        {localization.Intro.welcome}
-                    </Text>
-                    <Text className={styles.firstSectionSubtitle} block>
-                        {localization.Intro.fairlearnDashboard}
-                    </Text>
-                    <Text variant={'large'} block>
-                        {localization.Intro.introBody}
-                    </Text>
+                    <Stack wrap horizontalAlign={'start'} className={styles.firstSectionContainer}>
+                        <Text className={styles.firstSectionTitle} block>
+                            {localization.Intro.welcome}
+                        </Text>
+                        <Text className={styles.firstSectionSubtitle} block>
+                            {localization.Intro.fairlearnDashboard}
+                        </Text>
+                        <Text className={styles.firstSectionBody} variant={'large'} block>
+                            {localization.Intro.introBody}
+                        </Text>
+                        <div className={styles.firstSectionGraphics}>
+                            <svg width="358" height="156" viewBox="0 0 358 156" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it was discussed having svgs in an assets folder, but that can happen later. Would be nice to have the color not hardcoded but set by the theme.